src/components/NamespaceDropdown.tsx

```diff
@@ -3,40 +3,34 @@ import { connect } from 'react-redux';
 import { ThunkDispatch } from 'redux-thunk';
 import _ from 'lodash';
 import { style } from 'typestyle';
-import { Button, FormControl, InputGroup, OverlayTrigger, Popover } from 'patternfly-react';
+import { Button, ButtonVariant, Dropdown, DropdownToggle, TextInput } from '@patternfly/react-core';
 import { KialiAppState } from '../store/Store';
 import { activeNamespacesSelector, namespaceFilterSelector, namespaceItemsSelector } from '../store/Selectors';
 import { KialiAppAction } from '../actions/KialiAppAction';
 import { NamespaceActions } from '../actions/NamespaceAction';
 import NamespaceThunkActions from '../actions/NamespaceThunkActions';
 import Namespace from '../types/Namespace';
-import { PfColors } from './Pf/PfColors';
 import { HistoryManager, URLParam } from '../app/History';
 import {
   BoundingClientAwareComponent,
   PropertyType
 } from './BoundingClientAwareComponent/BoundingClientAwareComponent';
 import { KialiIcon } from 'config/KialiIcon';
-import TourStopContainer from './Tour/TourStop';
-import { GraphTourStops } from 'pages/Graph/GraphHelpTour';
```

Comment:
You need to add back the TourStop for the graph tour.

Reply:
Ah, right!